jobs/e2e_node/arm/init.yaml

```diff
@@ -0,0 +1,28 @@
+#cloud-config
+
+runcmd:
+  - echo "Test run from /tmp folder, remounting it"
+  - mount /tmp /tmp -o remount,exec,suid
+
```

Comment:
Not sure if having empty lines across a YAML list is supported.

Reply:
The empty lines will be ignored by the parser. I have tested it without any issue.